### PR TITLE
chore: update copy around definitions

### PR DESCRIPTION
**Pull request by tayloramurphy**

### CODE REVIEW

File: src/templates/Plugins.vue [hunk 1]
```diff
@@ -38,38 +38,31 @@
...
               <span v-if="$page.plugins.pluginType === 'extractor'">
                 pulls data from
                 <a :href="$page.plugins.domain_url">{{ $page.plugins.label }}</a> that can then be
                 sent to a destination using a <g-link to="/loaders">loader</g-link>.</span
               >
               <span v-if="$page.plugins.pluginType === 'loader'">
-                sends <g-link to="/extractors">extracted</g-link> data to a
-                <a :href="$page.plugins.domain_url">{{ $page.plugins.label }}</a>
-                destination.</span
+                sends data to into <a :href="$page.plugins.domain_url">{{ $page.plugins.label }}</a>
```
Comment (rwfeather) [inline src/templates/Plugins.vue:49]: "sends data to into _"

**Review by rwfeather — APPROVED** [review]
lgtm 👍 

if you're running the project locally, you can use `yarn format:write` to clear up the CI failure

**Review by cjohnhanson — APPROVED** [review]
Pending resolution of @rwfeather 's comments, this looks good to me.

File: src/templates/Plugins.vue (hunk 1 above)
Comment (tayloramurphy) [reply src/templates/Plugins.vue:49]:
```suggestion
                sends data into <a :href="$page.plugins.domain_url">{{ $page.plugins.label }}</a>
```

**Pull request merged**